fix: mobile toc issue with custom banners (#3382)
withastro/starlight: .changeset/weak-spiders-itch.md

---
'@astrojs/starlight': patch
---

> Fixes an issue where the mobile table of contents is unable to find the first heading when the banner of the page is higher than expected.

Reviewer: Small tweak here for clarity:

```suggestion
Fixes an issue where the mobile table of contents is unable to find the first heading when a page has a tall banner.
```

Most specifically: “taller” rather than “higher” because “higher” would normally just indicate a position (“The sixth-floor window is **higher** than the tree tops”)  vs dimensions of an item (“The tree is **taller** than the building”)
